src/frontend/src/components/forms/AuthFormOptions.tsx

```diff
@@ -19,12 +19,16 @@ export function AuthFormOptions({
       <Group>
         <ColorToggle />
         <LanguageToggle />
-        <Tooltip label={hostname}>
-          <IconServer onClick={toggleHostEdit} />
-        </Tooltip>
-        <Text c={'dimmed'}>
-          {server.version} | {server.apiVersion}
-        </Text>
+        {window.INVENTREE_SETTINGS.show_server_selector && (
+          <>
+            <Tooltip label={hostname}>
+              <IconServer onClick={toggleHostEdit} />
+            </Tooltip>
+            <Text c={'dimmed'}>
+              {server.version} | {server.apiVersion}
+            </Text>
```

Comment:
Suggestion: I think that versions should still be shown even if the sector is disabled

Reply:
We definitely can, but though I thought the current ui also doesn't show them. And there is one little bug which I was not able to fix. If you open the page with a clean local storage (e.g. private tab) the server version is not shown, you need to reload the page so that the hostList is loaded from local storage. Do you know how to fix that?

Reply:
Probably because the state is not fetched. I will submit a path to your branch.

Reply:
Found how to fix it in 5bd3782bad22d1adb8928f5a35b788fa252078ef